source/components/molecules/TimeSlotPicker/TimeSlotPicker.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { View, Text } from "react-native";
+import { ScrollView } from "react-native-gesture-handler";
+import stringify from "json-stable-stringify";
+import { DayPicker, TimeSpanButton } from "..";
+
+interface TimeSlotPickerProps {
+  value: any;
```

Comment:
Can't you specify a type for value and availableTimes?

Reply:
Fixed.